src/plugins/qmakeprojectmanager/qmakeparsernodes.h

```diff
@@ -234,6 +234,7 @@ class QMAKEPROJECTMANAGER_EXPORT QmakePriFile
     QSet<Utils::FileName> m_recursiveEnumerateFiles; // FIXME: Remove this?!
     QSet<QString> m_watchedFolders;
     bool m_includedInExactParse = true;
+    QMap<QString, QStringList> m_wildcardDirectoryContents;
```

Comment:
It looks so lonely without the m_wildcardWatcher here - shouldn't we move it as well? And the associated code?

Reply:
That is a good question. I'll have to think about it.

Reply:
Thinking about it I came to the conclusion that moving this to QmakePriFile was a wrong move. The results from expanding the wildcards are stored in the pro file level. So when a file removal is attempted in a pri file, the wildcards should be checked from the pro file.